package.json

```diff
@@ -65,6 +64,7 @@
     "vitest": "^0.33.0"
   },
   "dependencies": {
+    "@junobuild/analytics": "^0.0.9",
```

Comment:
Why is it no longer a `devDependencies`?

Reply:
No particular reason. Redo.